DSCResources/VE_XD7Controller/VE_XD7Controller.psm1

```diff
@@ -30,18 +30,19 @@ function Get-TargetResource {
 
     } #end begin
     process {
-
+        #Running scriptblock without invoke-command, so no need for $using:
```

Comment:
We still need to use `$using:` and PSRemoting for v4 compatibility and therefore, they need to be put back.